core/src/main/java/org/elasticsearch/cluster/routing/OperationRouting.java

```diff
@@ -199,12 +199,20 @@ private ShardIterator preferenceActiveShardIterator(IndexShardRoutingTable index
                 case LOCAL:
                     return indexShard.preferNodeActiveInitializingShardsIt(Collections.singleton(localNodeId));
                 case PRIMARY:
+                    deprecationLogger.deprecated("[_primary] has been deprecated in 6.0+, and will be removed in 7.0; " +
```

Comment:
The version here should be 6.1 (we are going to backport this to 6.x but not 6.0).